scripts/release/asset_publish.py

```diff
@@ -267,8 +267,7 @@ def _str2bool(v: str) -> bool:
         assets_by_type[asset.type.value].append(asset)
 
     # mlclient for the registry
-    credential = DefaultAzureCredential()
-    logger.print(f"{credential.get_token('https://management.azure.com/.default')}")
+    credential = AzureCliCredential()
     mlclient = MLClient(credential=credential, registry_name=registry_name)
```

Comment:
Please also consider:
```suggestion
    mlclient = MLClient(credential=credential, registry_name=registry_name, show_progress=False)
```